### ISSUE TITLE
Shows old SSID s but not the correct ones 

### ISSUE THREAD

**Issue by swordapi** [kalanda/esp8266-sniffer#21]
![Screenshot_20220615-164145_EspTerm.png](https://user-images.githubusercontent.com/70171212/173855389-f55592cc-807b-4fcf-b934-df0e299fd460.png)

These Networks don't exist. And it shows old names of the esp wifi. Any solutions?

**Comment by user011000001** [comment]
I think it shows ssids that devices are looking for. May I am wrong but can be

**Comment by kalanda** [comment]
> I think it shows ssids that devices are looking for. May I am wrong but can be

you are right, those SSIDs are SSIDs which devices around you, are looking for because was connected to in the past. Is explained at the home of this project https://github.com/kalanda/esp8266-sniffer?tab=readme-ov-file#introduction
